app/assets/stylesheets/modules/markers.css.scss

```diff
@@ -9,6 +9,10 @@
   color: transparent;
   cursor: pointer;
 
+  &.measurement {
+    background-color: pink;
```

Comment:
Is it supposed to be pink? 🤔

Reply:
should be `$blue`

Reply:
also `measurement` is not intuitive for me, I'm thinking sth like `trace-marker`, sth that says that this indicates the exact measurement in the session, maybe `current-measurement`, I don't know. any ideas?

Reply:
Discuss on call